### PR TITLE
feat: Add static directions table to trips

### PR DESCRIPTION
**Pull request by rymarczy**
At the request of OPMI, this PR adds `directions` table data from the GTFS static schedule to the `vehicle_trips` table.

The directions data is added by creating a `static_diretions` table in the RDS, and then joining the `direction` and `direction_destination` fields to newly processed trips with the `update_directions` function in `l1_rt_trips.py`

Asana Task: https://app.asana.com/0/1204517240145658/1203464608464154

### CODE REVIEW

File: python_src/src/lamp_py/migrations/versions/performance_manager/001_be4c10c548f0_initial_rds_schema.py
Comment (rymarczy) [inline python_src/src/lamp_py/migrations/versions/performance_manager/001_be4c10c548f0_initial_rds_schema.py:1]: renamed all these to make them look a lot nicer in the folder browser

File: python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py [hunk 1]
```diff
@@ -202,6 +221,7 @@ def get_table_objects() -> Dict[str, StaticTableDetails]:
         "trips": trips,
         "calendar": calendar,
         "calendar_dates": calendar_dates,
+        "directions": directions,
```
Comment (mzappitello) [inline python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py:224]: we don't care about the order that directions are loaded in do we?

**Review by mzappitello — COMMENTED** [review]
two comments. one on git practices and one is a question.

File: python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py (hunk 1 above)
Comment (rymarczy) [reply python_src/src/lamp_py/performance_manager/l0_gtfs_static_load.py:224]: We do not, the table is not used until RT trips records are loaded.

**Comment by mzappitello** [comment]
🍰

**Pull request merged**